site/map.js

```diff
@@ -135,7 +145,7 @@ function makePopup(feature) {
     // Based on https://stackoverflow.com/questions/10889954
     var properties = feature.properties;
     var desc = "";
-
+    
```

Comment:
There should be a "strip trailing spaces" command or similar you can run to fix things like this